test/unit/Rolls.t.sol

```diff
@@ -224,13 +226,18 @@ contract RollsTest is BaseAssetPairTestSetup {
 
         // Check new provider position details
         CollarProviderNFT.ProviderPosition memory newProviderPos = providerNFT.getPosition(newProviderId);
+        assertEq(newProviderPos.offerId, nextProviderOfferId);
+        assertEq(newProviderPos.takerId, newTakerId);
         assertEq(newProviderPos.duration, duration);
         assertEq(newProviderPos.expiration, block.timestamp + duration);
         assertEq(newProviderPos.providerLocked, expected.newProviderLocked);
         assertEq(newProviderPos.putStrikePercent, ltv);
         assertEq(newProviderPos.callStrikePercent, callStrikePercent);
         assertFalse(newProviderPos.settled);
         assertEq(newProviderPos.withdrawable, 0);
+
+        // check miLocked for offer was 0
```

Comment:
do we care that this would also pass if the offer didnt exist? (other things prolly would be broken so wouldnt reach this but yeah)

I know it's never gonna be a value other than 0 but if we care to "check it exists" i dont think this covers it, as farfetched as it would be for it not to 😆

Reply:
Yeah good point. But since a bit above it the position is checked to be created from that offer it's fine.

> `assertEq(newProviderPos.offerId, nextProviderOfferId);`